backend/Models/Data/Sighting.cs

```diff
@@ -13,7 +13,7 @@ public class Sighting
     [ForeignKey("User")]
     public int UserId { get; set; }
     public User User { get; set; } = null!;
-    public int SpeciesId { get; set; }
+    public int WhaleSpeciesId { get; set; }
```

Comment:
I think that this was changed to SpeciesId in a previous PR due to something related to the foreign key (I can't quite remember) but it's best to keep it as SpeciesId for now unless that's causing problems?

Reply:
This was causing problems because we couldn't use the existing /sightings/create endpoint to create a new sighting, I think the error was something like, it couldn't find the field SpeciesId in the database, because in the database it is still called WhaleSpeciesId @Kiran-R-K